### DISCUSSION TITLE
AttributeError: module 'customtkinter' has no attribute 'CTk'

### DISCUSSION THREAD

**Discussion by Vikasitha444** [TomSchimansky/CustomTkinter#1081] [General]
I copied and paste the complex_example.py from the examples folder and I got this error (**AttributeError: module 'customtkinter' has no attribute 'CTk'**) again and again. I tried to solve this by updating pip and updating the custom-Tkinter library. But nothing happened. Please help me to fix this.

**Comment by its-mr-gray** [comment]
@Vikasitha444  https://github.com/TomSchimansky/CustomTkinter/issues/447 refer to the comment by @Akascape. had the same error myself and got it working.

**Comment by Vikasitha444** [comment]
**Traceback (most recent call last):
  File "C:\Users\Vikasitha\Documents\other.py", line 7, in <module>
    app = customtkinter.CTk()  # create CTk window like you do with the Tk window
AttributeError: module 'customtkinter' has no attribute 'CTk'**


Still, I'm getting the same error even though I renamed my .py file as "other.py". And also I've got this error too.

**AttributeError: module 'customtkinter' has no attribute 'set_default_color_theme'**

I think, name CTk() has changed to different keyword.  Please send me an example with new version of customtkinter module.

**Reply by hemanthreddygangula** [reply]
I too had the same problem.
Try installing 'customtkinter' version of 4.6.3
It will solve the problem.

**Comment by 1mkRE** [comment]
Which example you try exactly the original from Github or a code example from any chat??


Please check how you imported cumstomtkinter.

1)
import customtkinter

......
......

class App(customtkinter.CTk):

........

2)
from customtkinter import *

..............

class App(CTk):
.................

**Comment by planetminguez** [comment]
guys I just deleted customtkinter and reinstalled it from official repo.....there had to be something wrong with the naming cuz it works now. 

Step 1 --> pip3 uninstall customtkinter
Step 2 -->  pip3 install customtkinter

I am pretty sure I installed it from Pypi the 1st time
